js/widgets/fixedToolbar.js

```diff
@@ -185,39 +170,42 @@ define( [ "jquery", "../widget", "../core", "../animationComplete", "../navigati
 		},
 
 		show: function( notransition ) {
-			var hideClass = "ui-fixed-hidden",
+			var hideClass = "ui-toolbar-fixed-hidden",
+				self = this,
 				$el = this.element;
 
 			if ( this._useTransition( notransition ) ) {
-				$el
-					.removeClass( "out " + hideClass )
-					.addClass( "in" )
-					.animationComplete(function () {
-						$el.removeClass( "in" );
-					});
+				this._removeClass( null, "out" );
+				this._removeClass( hideClass );
+				this._addClass( null, "in" );
+				$el.animationComplete( function() {
+					self._removeClass( null, "in" );
+				} );
 			}
 			else {
-				$el.removeClass( hideClass );
+				this._removeClass( hideClass );
 			}
 			this._visible = true;
 		},
 
 		hide: function( notransition ) {
-			var hideClass = "ui-fixed-hidden",
+			var hideClass = "ui-toolbar-fixed-hidden",
```

Comment:
I don't think there is any point in saving this to a var seems like its only used once?